.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ executors:
       - image : prestocpp/velox-check:mikesh-20210609
```

Comment:
Just curious as to why we don't have to update this image?

Reply:
This is a different image with just enough stuff installed to run `make format-check`. It's not used to build/test. It's not even using Centos.

This image was split out because we needed a specific clang version that was already available on ubuntu hirsute, and building that version in the centos8 container took forever.